transport-native-epoll/src/main/c/netty_epoll_native.c

```diff
@@ -141,6 +137,15 @@ static void netty_epoll_native_eventFdRead(JNIEnv* env, jclass clazz, jint fd) {
     }
 }
 
+static void netty_epoll_native_timerFdRead(JNIEnv* env, jclass clazz, jint fd) {
+    uint64_t timerFireCount;
+
+    if (read(fd, &timerFireCount, sizeof(uint64_t)) < 0) {
```

Comment:
not to stick my nose in, but technically this can return fewer than 8 bytes right?